tensilelite/Tensile/TensileCreateLibrary.py

```diff
@@ -835,8 +835,12 @@ def buildObjectFileNames(kernelWriterAssembly, kernels, kernelHelperObjs):
   asmKernels = (k for k in kernels if k['KernelLanguage'] == 'Assembly')
 
   # Build a list of kernel object names.
+  # Asm based kernels target the configured ISA
+  asmArchs = collections.defaultdict(list)
   for kernel in asmKernels:
-    asmKernelNames += [kernelWriterAssembly.getKernelFileBase(kernel)]
+    kernelName = kernelWriterAssembly.getKernelFileBase(kernel)
+    asmKernelNames += [kernelName]
```

Comment:
```suggestion
    asmKernelNames.append(kernelName)
```
Not that this will make a huge impact, but it is considered a more efficient approach than a list concatenation for a single element.